pkg/kine/drivers/dqlite/driver.go

```diff
@@ -22,21 +24,41 @@ func init() {
 	}
 }
 
-type Driver = sqlite.Driver
+type Driver struct {
+	*sqlite.Driver
+
+	goDqliteApp *app.App
+}
+
+func (d *Driver) Compact(ctx context.Context, revision int64) (err error) {
+	// Skip the compaction if we're not the leader.
+	//
+	// WARNING: overridden methods will not be visible from the embedded
+	// (sqlite) driver.
+	isLeader, err := isLocalNodeLeader(ctx, d.goDqliteApp)
+	if err != nil {
+		logrus.WithError(err).Warning("couldn't determine if the local node has leader role, allowing the compaction to proceed")
```

Comment:
```suggestion
		logrus.WithError(err).Warning("Couldn't determine whether the local node is the leader, allowing the compaction to proceed")
```
-> We have to capitalize to stay consistent with the other warnings in this repo.